apps/predbat/predbat.py

```diff
@@ -2765,15 +2765,21 @@ def mimic_target_soc(self, current_charge_limit):
         Returns:
             None
         """
-        if self.soc_percent >= float(current_charge_limit):
+        charge_power = self.base.get_arg("charge_rate", index=self.id, default=2600.0)
```

Comment:
What does this do exactly, as the entity 'charge_rate' is created by Predbat itself won't it just read back the original max setting?

Reply:
Oh I understand now from your comments